packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -232,7 +232,7 @@ export namespace RecordingTestMetadataV3 {
   export type UserActionEventStack = Array<{
     columnNumber: number;
     fileName: string;
-    functionName: string;
+    functionName?: string;
```

Comment:
Why?

Reply:
The playwright type has `function` as `string | undefined` so I've propagated that.